### PR TITLE
Handle all the store boot properties

### PR DESCRIPTION
**Pull request by nthanvi**
Handle all the gemfirexd store boot properties
minor changes for scala code styling

### CODE REVIEW

File: snappy-tools/src/main/scala/io/snappydata/ToolsCallbackImpl.scala [hunk 1]
```diff
@@ -37,16 +37,21 @@ object ToolsCallbackImpl extends ToolsCallback {
...
-            "be provided in the format host[port] or host:port", null)
-      }
-      properties.setProperty("locators", locator)
-      sc.getConf.getOption(Property.mcastPort).map(
-        properties.setProperty("mcast-port", _))
-    }.getOrElse(properties.setProperty("mcast-port",
-      sc.getConf.get(Property.mcastPort)))
+    sc.getConf.getAll.filter(
+      p => p._1.startsWith(Constant.STORE_PROPERTY_PREFIX)).
+        foreach(p => p._1 match {
```
Comment (hbhanawat) [inline snappy-tools/src/main/scala/io/snappydata/ToolsCallbackImpl.scala]: Use different names of the variables for the inner and outer blocks.

Comment (hbhanawat) [inline snappy-tools/src/main/scala/io/snappydata/ToolsCallbackImpl.scala]: okay they are different blocks. But changing name would still be good from reading perspective.

**Comment by hbhanawat** [comment]
Looks good. But we wanted an overall strategy for handling properties for job-server, spark and gemxd.

**Comment by nthanvi** [comment]
@hbhanawat  - SNAP-322 - enables user to pass all the gemxd store related properties from the spark conf file to gemxd. if you think we need strategy/handling for job server and spark properties as well then please open respective jira to track those gaps.

**Pull request merged**